packages/core/src/interfaces/events.ts

```diff
@@ -64,10 +64,10 @@ export enum Axis {
  * enum of all area graph events
  */
 export enum Area {
-	AREA_MOUSEOVER = 'area-mouseover',
-	AREA_MOUSEMOVE = 'area-mousemove',
-	AREA_CLICK = 'area-click',
-	AREA_MOUSEOUT = 'area-mouseout',
+	AREA_MOUSEOVER = 'scatter-mouseover',
+	AREA_MOUSEMOVE = 'scatter-mousemove',
+	AREA_CLICK = 'scatter-click',
+	AREA_MOUSEOUT = 'scatter-mouseout',
```

Comment:
we should probably call these `POINT_***` since the events are actually related to the points in the area chart